shadows/supportv4/build.gradle

```diff
@@ -8,45 +8,43 @@ configurations {
     earlyTestRuntime
 }
 
-def supportLibraryVersions = "28.0.0"
```

Comment:
Maybe we don't need update supportv4, because it is scheduled to removed totally at 4.9.